src/runtime/fetcher.js

```diff
@@ -40,6 +40,14 @@ export class Fetcher {
    * @param {!string|!Object} unusedData
    */
   sendBeacon(unusedUrl, unusedData) {}
+
+  /**
+   * POST data to a URL endpoint, get a Promise for a response
+   * @param {!string} unusedUrl
+   * @param {!string|!Object} unusedMessage
+   * @return {!Promise<!../utils/xhr.FetchResponse>}
+   */
+  sendMessage(unusedUrl, unusedMessage) {}
```

Comment:
I think `sendPost` would be a more descriptive name, what do you think? Also, would it be possible to add a test for this and create a new PR for it?

Reply:
I'm only just getting to the tests.  I can break things up a bit into separate PRs though

Reply:
Sweet!